### PR TITLE
Use tested packages

### PR DESCRIPTION
**Pull request by maelk**
Reverts metal3-io/ironic-image#154
Reverts the revert to use the intended packages after ironic inspector bug fix

### CODE REVIEW

File: configure-ironic.sh [hunk 1]
```diff
@@ -39,6 +38,9 @@ fast_track = ${IRONIC_FAST_TRACK}
 
 [inspector]
 endpoint_override = http://${IRONIC_URL_HOST}:5050
+
+[service_catalog]
+endpoint_override = http://${IRONIC_URL_HOST}:6385
```
Comment (juliakreger) [inline configure-ironic.sh:43]: Can I get some more detail regarding why this is needed. It seems like a defect upstream that we need to fix.

Comment (maelk) [reply configure-ironic.sh:43]: It comes from https://docs.openstack.org/releasenotes/ironic/unreleased.html, api_url was deprecated, so ironic was unable to find the service endpoint without that and failing on contacting keystone

Comment (maelk) [reply configure-ironic.sh:43]: The stack trace was :
```
2020-05-14 12:57:28.897 27 ERROR ironic.conductor.cleaning [req-0295b6c3-6f76-4d97-a2c8-0325fd625344 - - - - -] Failed to prepare node a1758701-211c-4394-9fa8-775b36edf6cf for cleaning: Couldn't get the URL of the Ironic API service from the configuration file or keystone catalog. Keystone error: Service type baremetal with endpoint type internal not found in keystone service catalog.: ironic.common.exception.InvalidParameterValue: Couldn't get the URL of the Ironic API service from the configuration file or keystone catalog. Keystone error: Service type baremetal with endpoint type internal not found in keystone service catalog.
2020-05-14 12:57:28.897 27 ERROR ironic.conductor.cleaning Traceback (most recent call last):
2020-05-14 12:57:28.897 27 ERROR ironic.conductor.cleaning   File "/usr/lib/python3.6/site-packages/ironic/drivers/modules/deploy_utils.py", line 96, in get_ironic_api_url
2020-05-14 12:57:28.897 27 ERROR ironic.conductor.cleaning     ironic_api = keystone.get_endpoint('service_catalog', **adapter_opts)
2020-05-14 12:57:28.897 27 ERROR ironic.conductor.cleaning   File "/usr/lib/python3.6/site-packages/ironic/common/keystone.py", line 124, in get_endpoint
2020-05-14 12:57:28.897 27 ERROR ironic.conductor.cleaning     service_type=service_type, endpoint_type=endpoint_type)
2020-05-14 12:57:28.897 27 ERROR ironic.conductor.cleaning ironic.common.exception.CatalogNotFound: Service type baremetal with endpoint type internal not found in keystone service catalog.
2020-05-14 12:57:28.897 27 ERROR ironic.conductor.cleaning 
2020-05-14 12:57:28.897 27 ERROR ironic.conductor.cleaning During handling of the above exception, another exception occurred:
2020-05-14 12:57:28.897 27 ERROR ironic.conductor.cleaning 
2020-05-14 12:57:28.897 27 ERROR ironic.conductor.cleaning Traceback (most recent call last):
2020-05-14 12:57:28.897 27 ERROR ironic.conductor.cleaning   File "/usr/lib/python3.6/site-packages/ironic/conductor/cleaning.py", line 96, in do_node_clean
2020-05-14 12:57:28.897 27 ERROR ironic.conductor.cleaning     prepare_result = task.driver.deploy.prepare_cleaning(task)
2020-05-14 12:57:28.897 27 ERROR ironic.conductor.cleaning   File "/usr/lib/python3.6/site-packages/ironic_lib/metrics.py", line 59, in wrapped
2020-05-14 12:57:28.897 27 ERROR ironic.conductor.cleaning     result = f(*args, **kwargs)
2020-05-14 12:57:28.897 27 ERROR ironic.conductor.cleaning   File "/usr/lib/python3.6/site-packages/ironic/drivers/modules/agent.py", line 679, in prepare_cleaning
2020-05-14 12:57:28.897 27 ERROR ironic.conductor.cleaning     task, manage_boot=CONF.agent.manage_agent_boot)
2020-05-14 12:57:28.897 27 ERROR ironic.conductor.cleaning   File "/usr/lib/python3.6/site-packages/ironic/drivers/modules/deploy_utils.py", line 593, in prepare_inband_cleaning
2020-05-14 12:57:28.897 27 ERROR ironic.conductor.cleaning     ramdisk_opts = build_agent_options(task.node)
2020-05-14 12:57:28.897 27 ERROR ironic.conductor.cleaning   File "/usr/lib/python3.6/site-packages/ironic/drivers/modules/deploy_utils.py", line 543, in build_agent_options
2020-05-14 12:57:28.897 27 ERROR ironic.conductor.cleaning     'ipa-api-url': get_ironic_api_url(),
2020-05-14 12:57:28.897 27 ERROR ironic.conductor.cleaning   File "/usr/lib/python3.6/site-packages/ironic/drivers/modules/deploy_utils.py", line 103, in get_ironic_api_url
2020-05-14 12:57:28.897 27 ERROR ironic.conductor.cleaning     "%s") % str(e))
2020-05-14 12:57:28.897 27 ERROR ironic.conductor.cleaning ironic.common.exception.InvalidParameterValue: Couldn't get the URL of the Ironic API service from the configuration file or keystone catalog. Keystone error: Service type baremetal with endpoint type internal not found in keystone service catalog.
2020-05-14 12:57:28.897 27 ERROR ironic.conductor.cleaning 
2020-05-14 12:57:28.992 27 ERROR ironic.conductor.task_manager [req-0295b6c3-6f76-4d97-a2c8-0325fd625344 - - - - -] Node a1758701-211c-4394-9fa8-775b36edf6cf moved to provision state "clean failed" from state "cleaning"; target provision state is "available": ironic.common.exception.InvalidParameterValue: Couldn't get the URL of the Ironic API service from the configuration file or keystone catalog. Keystone error: Service type baremetal with endpoint type internal not found in keystone service catalog.
```

Comment (maelk) [reply configure-ironic.sh:43]: @juliakreger as far as I can say, this is expected behaviour, since api_url was deprecated, isn't it ?

Comment (juliakreger) [reply configure-ironic.sh:43]: Ahh, that issue. And master branch support, I believe was removed. *sigh* I fought against that for so long it is not even funny. Thanks for the additional detail.

**Pull request merged**